docs/scripting-ref.rst

```diff
@@ -2230,3 +2230,128 @@ Trigger mouse release event in web page.
 **Async:**: no.
 
 See notes about mouse events in :ref:`splash-mouse-click`.
+
+.. _splash-send-keys:
+
+splash:send_keys
+----------------
+
+Send keyboard events to page context.
+
+**Signature:** ``splash:send_keys(keys)``
+
+**Parameters**
+
+* keys - string representing the keys to be sent as keyboard events.
+
+**Returns:** nil
+
+**Async:**: no.
+
+Key sequences are specified by using a small subset of emacs edmacro syntax:
+
+* whitespace is ignored and only used to separate the different keys
+* characters are literally represented
+* ``RET``, ``SPC``, ``TAB``, ``DEL``, ``ESC`` are reserved and represent
+  special control characters.
+* words within brackets represent function keys, like ``<Return>``, ``<Left>``
+  or ``<Home>``. See `Qt docs`__ for a full list of function keys. ``<Foo>``
+  will try to match ``Qt::Key_Foo``.
```

Comment:
My vote is to remove support for RET, DEL, etc. It should make validation easier, we'll be able to detect typos and raise errors. Alternatively, we could support full edmacro format, I'm also fine with that.